js/src/dapps/registry/Names/names.js

```diff
@@ -78,35 +79,21 @@ const renderQueue = (queue) => {
 class Names extends Component {
 
   static propTypes = {
+    error: nullableProptype(PropTypes.object.isRequired),
```

Comment:
As per comment above, believe that `nullableProptype(type).isRequired` is probably more in line with intent